### ISSUE TITLE
Problem with RetentionPolicyConfiguration

### ISSUE THREAD

**Issue by tagantroy** [cdsap/Talaiot#76]
I checked https://github.com/cdsap/Talaiot/blob/master/talaiot/src/main/kotlin/com/cdsap/talaiot/configuration/InfluxDbPublisherConfiguration.kt
it looks like RetentionPolicyConfiguration is not correct and all fields should be vars instead of vals. 
I tried to set a custom retention policy
```
talaiot {
    publishers {
        influxDbPublisher {
            dbName = "xxxxxx"
            url = "xxxxxx"
            username = "xxxxxx"
            password = "xxxxxx"
            taskMetricName = "task"
            buildMetricName = "build"
            retentionPolicyConfiguration {
                name = "rpTalaiot"
                duration = "4w"
                shardDuration = "30m"
                replicationFactor = 1
                isDefault = true
            }
        }
    }
}
```
and build failing with 
```
> Could not find method retentionPolicyConfiguration() for arguments [build_duu1x1uzp5fcat966lugpczfc$_run_closure1$_closure4$_closure6$_closure7@443375b6] on object of type com.cdsap.talaiot.TalaiotExtension.
```

**Comment by cdsap** [comment]
Hi @tagantroy, thank you very much for the reporting. Indeed, you're right. A hotfix has been released under 1.0.1. You can check it

**Comment by tagantroy** [comment]
Hello @cdsap thanks for a quick fix. It works in 1.0.1.
